build/parseSpec.c

```diff
@@ -385,7 +385,11 @@ static int readLineFromOFI(rpmSpec spec, OFI_t *ofi)
 retry:
     /* Make sure the current file is open */
     if (ofi->fp == NULL) {
-	ofi->fp = fopen(ofi->fileName, "r");
+	if (!strcmp(ofi->fileName, "-")) {
```

Comment:
I loathe the `!strcmp(...)` use, it's soo easy to misread despite knowing the trick. Please use either explicit `strcmp(...) == 0` or  `rstreq()` for comparing string equivalence.

Reply:
Here you go!